### PR TITLE
Details addition

### PR DESCRIPTION
**Pull request by c0dzilla**
Added Nikhil Kaushik to team.

### CODE REVIEW

File: _data/team.yml [hunk 1]
```diff
@@ -374,9 +374,19 @@
...
+    Email: mailto:nikhil.celtics@gmail.com
+    StackOverflow: http://stackexchange.com/users/9712323/nikhil-kaushik
+
 - name: Kartikey Pandey
   pic: kartik.jpeg
   links:
     Facebook: https://www.facebook.com/kartikey.pandey.900
     Email: mailto:k.p.2511998@gmail.com
-    GitHub: https://github.com/KP251
+    GitHub: https://github.com/KP25
```
Comment (uttu357) [inline _data/team.yml]: Why did you change kartikey's username? Fix this..

File: _data/team.yml [hunk 2]
```diff
@@ -382,11 +389,4 @@
...
     StackOverflow: http://stackexchange.com/users/9712323/nikhil-kaushik
 
-- name: Kartikey Pandey
-  pic: kartik.jpeg
-  links:
-    Facebook: https://www.facebook.com/kartikey.pandey.900
-    Email: mailto:k.p.2511998@gmail.com
-    GitHub: https://github.com/KP25
-
 1
```
Comment (uttu357) [inline _data/team.yml]: Remove this 1

**Comment by uttu357** [comment]
Also resolve the merge conflicts by performing a rebase..

**Comment by uttu357** [comment]
@c0dzilla Status?

**Comment by c0dzilla** [comment]
@uttu357 I think everything is fine now.
